### PR TITLE
Replace Interpolation With Fstring

### PR DESCRIPTION
**Pull request by girisagar46**
I want to merge this change because...

- I want to replace string interpolation using F strings.
- There is a mixed usage of `%-formatting` and `Fstrings`, so I want to unify as much as possible.
- However, I did not touch the graphQL query interpolation templates

# Impact

None.

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized, and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

File: saleor/app/tasks.py [hunk 1]
```diff
@@ -22,25 +22,25 @@ def install_app_task(job_id, activate=False):
         msg = ", ".join([f"{name}: {err}" for name, err in e.message_dict.items()])
         app_installation.message = msg
     except AppInstallationError as e:
-        logger.warning("Failed to install app. Error: %s", e)
+        logger.warning(f"Failed to install app. Error: {e}")
```
Comment (8r2y5) [inline saleor/app/tasks.py:25]: F-string is not suitable for logging, please revert to lazy formatting.

**Review by 8r2y5 — CHANGES_REQUESTED** [review]

File: saleor/app/tasks.py (hunk 1 above)
Comment (girisagar46) [reply saleor/app/tasks.py]: Ah! Sorry, I didn't know that. Looks like it's still in debate and has an open issue https://github.com/python/cpython/issues/90358 for it. 
Thank you, @8r2y5, for letting me know. I've reverted the f-strings in log messages here: 67e6954

File: saleor/account/i18n.py [hunk 2]
```diff
@@ -183,9 +180,8 @@ def validate_address(self, data):
         try:
             data["country_code"] = data.get("country", "")
             if data["street_address_1"] or data["street_address_2"]:
-                data["street_address"] = "%s\n%s" % (
-                    data["street_address_1"],
-                    data["street_address_2"],
+                data["street_address"] = (
+                    f'{data["street_address_1"]}\n' f'{data["street_address_2"]}'
```
Comment (8r2y5) [inline saleor/account/i18n.py]: This could be one string
```suggestion
                    f'{data["street_address_1"]}\n{data["street_address_2"]}'
```

**Review by 8r2y5 — APPROVED** [review]

**Review by kadewu — APPROVED** [review]

**Comment by girisagar46** [comment]
@maarcingebala, Could you please approve the CI workflows?

**Comment by maarcingebala** [comment]
It's all good, thank you for your contribution to Saleor @girisagar46 🎉

**Pull request merged**